fe/fe-core/src/test/java/org/apache/doris/rewrite/InferFiltersRuleTest.java

```diff
@@ -299,4 +299,25 @@ public void testWhereBetweenPredicate() throws Exception {
         Assert.assertTrue(planString.contains("`tb2`.`k1` >= 1"));
         Assert.assertTrue(planString.contains("`tb2`.`k1` <= 2"));
     }
+
+    @Test
```

Comment:
infer just occurs when predicate can pushdown.  I think this point should be included in unit test.

We should add `failed to infer predicate when can't pushdown` case.